src/main/java/org/jabref/gui/entryeditor/EntryEditor.java

```diff
@@ -1488,54 +1501,48 @@ private void warnDuplicateBibtexkey() {
     }
 
     private void warnEmptyBibtexkey() {
-        panel.output(Localization.lang("Empty BibTeX key") + ". " + Localization.lang("Grouping may not work for this entry."));
+        panel.output(Localization.lang("Empty BibTeX key") + ". "
+                + Localization.lang("Grouping may not work for this entry."));
     }
 
-
     public AbstractAction getNextEntryAction() {
         return nextEntryAction;
     }
 
-
     public AbstractAction getPrevEntryAction() {
         return prevEntryAction;
     }
 
-
     public SwitchLeftAction getSwitchLeftAction() {
         return switchLeftAction;
     }
 
-
     public SwitchRightAction getSwitchRightAction() {
         return switchRightAction;
     }
 
-
     public SaveDatabaseAction getSaveDatabaseAction() {
         return saveDatabaseAction;
     }
 
-
     public HelpAction getHelpAction() {
         return helpAction;
     }
 
-
     public GenerateKeyAction getGenerateKeyAction() {
         return generateKeyAction;
     }
 
-
     public StoreFieldAction getStoreFieldAction() {
         return storeFieldAction;
     }
 
-
     private class AutoLinkAction extends AbstractAction {
+
         public AutoLinkAction() {
             putValue(Action.SMALL_ICON, IconTheme.JabRefIcon.AUTO_FILE_LINK.getIcon());
-            putValue(Action.SHORT_DESCRIPTION, Localization.lang("Automatically set file links for this entry") + " (Alt-F)");
```

Comment:
If we did further code improvement, we would use the other way to show the shortcuts. Further, <kbd>Alt</kbd> + <kbd>F</kbd> is no longer the valid shortcut for that, is it?

Can you follow up in another PR?